patches/0032-avfilter-Add-H2H-support-in-tonemap_vaapi.patch

```diff
@@ -243,12 +243,16 @@ index cd2f4c2195..93946306db 100644
      return 0;
  }
  
-@@ -383,10 +553,11 @@ static const AVOption tonemap_vaapi_options[] = {
+@@ -386,10 +556,15 @@ static const AVOption tonemap_vaapi_options[] = {
      { "t",        "Output color transfer characteristics set",
        OFFSET(color_transfer_string),  AV_OPT_TYPE_STRING,
-       { .str = NULL }, .flags = FLAGS, "transfer" },
-+    { "display", "set master display",  OFFSET(master_display), AV_OPT_TYPE_STRING, {.str=NULL}, CHAR_MIN, CHAR_MAX, FLAGS },
-+    { "light",   "set content light",   OFFSET(content_light),  AV_OPT_TYPE_STRING, {.str=NULL}, CHAR_MIN, CHAR_MAX, FLAGS },
+       { .str = NULL }, .flags = FLAGS, .unit = "transfer" },
++    { "display", "set master display",
++      OFFSET(master_display), AV_OPT_TYPE_STRING,
++      { .str = NULL }, .flags = FLAGS, .unit = "display" },
++    { "light",   "set content light",
++      OFFSET(content_light),  AV_OPT_TYPE_STRING,
++      { .str = NULL }, .flags = FLAGS, .unit = "light" },
```

Comment:
How about ignore .unit of display and light? Since they are individual options. Others LGTM.

Reply:
Updated, thanks